tasks/ghost-os.yml

```diff
@@ -4,3 +4,9 @@
   template: src=etc/init/ghost.conf.j2 dest=/etc/init/ghost.conf
   notify: restart ghost
   tags: [ghost]
```

Comment:
You are right. http://linux.softpedia.com/blog/Ubuntu-15-04-Users-Can-Switch-Off-Systemd-and-Use-Upstart-479373.shtml

We'll keep the upstart file copied then and will add the condition if we find It necessary in the future.